cypress/support/pages/roles.ts

```diff
@@ -0,0 +1,47 @@
+import { keys } from "cypress/types/lodash";
+import { verify } from "cypress/types/sinon";
+import { randomNumber, role, smallRandomNumber, team, user } from "../../constants/globalConstants"
+
+//general data
+
+const name_1 = 'EA';
+const role_name = name_1+randomNumber;
+
+
+
+ //create a role
+Cypress.Commands.add('createARole', () => {
```

Comment:
so Cypress commands are usually reserved for actions that are performed multiple times at different locations across the app. The flow of creating a role is more of a single test case, than a command we'd want to reuse across multiple tests